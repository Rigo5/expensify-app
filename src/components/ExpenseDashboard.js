import React from 'react';
import ExpenseList from './expensesList';
import ExpenseListFilter from './ExpensesListFilter';
import ExpenseSummary from './ExpenseSummary';

const ExpenseDashboard = () => {
    return (
        <div>
            <p>la merda di cane puzza</p>
            <ExpenseListFilter />
            <ExpenseList />
            <ExpenseSummary />
        </div>
    );
}

export default ExpenseDashboard;