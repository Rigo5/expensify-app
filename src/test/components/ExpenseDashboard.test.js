import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboard from '../../components/ExpenseDashboard';

test('should expense dashboard render', () => {
    const wrapper = shallow(<ExpenseDashboard />);

    expect(wrapper).toMatchSnapshot();
})
