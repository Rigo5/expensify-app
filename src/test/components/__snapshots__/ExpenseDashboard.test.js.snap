// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should expense dashboard render 1`] = `
<div>
  <p>
    la merda di cane puzza
  </p>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
  <Connect(ExpenseSummary) />
</div>
`;
